doc/rbd/rbd-mirroring.rst

```diff
@@ -289,6 +294,16 @@ For example::
 .. tip:: You can enable journaling on all new images by default by adding
    ``rbd default features = 125`` to your Ceph configuration file.
 
+.. tip:: ``rbd-mirror`` tunables are set by default to values suitable for
+   mirroring an entire pool.  When using ``rbd-mirror` to migrate single
```

Comment:
s/``rbd-mirror`/``rbd-mirror``/

(the string "rbd-mirror" needs another backtick added after it so that it doesn't end up marking up the rest of the line)

Reply:
Who picked RST? and who decided to make this markup just different enough from Markdown to drive us crazy?